src/parse/tibasic.pegjs

```diff
@@ -429,4 +441,7 @@ Statement
   / Assignment
   / CtlStatement
   / IoStatement
-  / ValueStatement
\ No newline at end of file
+  / ValueStatement
+  
+  _ "whitespace"
+  = [ \t\n\r]*
```

Comment:
I don't think this is the right approach. Whitespace is TI-BASIC is meaningful, so while these are valid:
```
1+1
1 and 1
```

These are syntax errors:
```
1+ 1
1 and  1
```

This means we can't collapse whitespace into a single label this way, and we certainly can't include newlines in there.

I think for `and` and `or` the spaces can be built into the token itself, just like `If `.

Reply:
Agreed - my thinking was that an explicit whitespace declaration might make error handling easier, but I think your approach is as good and removes lots of repetition.